am-ui/webcomponents/create-ad-webcomponent/src/components/CreateAdFields.vue

```diff
@@ -6,8 +6,8 @@
       class="form-field full"
```

Comment:
Please no changes to create-ad-webcomponent in this PR

Reply:
Sorry, I think most of the changes is just the linter going at it since on `main` the google coding style isn't in the .eslintrc.json file (it is present on your branch though). If you want to I can add the google style to this branch too so the linting changes would be gone. 
The only actual change that is actually required to post ads required for the search-results is changing this:
```v-model="fieldValues[field.key]"```
to
```v-model="fieldValues[field.field_id]"```
in order to save the correct data to the database. Is this change fine? Otherwise we can't post results that can be displayed for the results.

Reply:
Ok yea please add the google style here too. And yes the field.key change is fine, in fact I think its an error from my side only the checkbox input is like that, should be field.field_id. But I change that in the post ad unit tests PR, and test it well after merging main with content of this PR before closing the other PR.